test/e2e/framework/framework_constants.go

```diff
@@ -44,7 +44,7 @@ const (
 	InstallAllPluginsFromGroupCmd       = "%s plugin install --group %s"
 	DescribePluginCmd                   = "%s plugin describe %s"
 	UninstallPLuginCmd                  = "%s plugin uninstall %s --yes"
-	CleanPluginsCmd                     = "%s plugin clean"
+	CleanPluginsCmd                     = "%s plugin clean --yes"
```

Comment:
e2e is failing because this is also used with a legacy CLI that doesn't support this flag.
cc @chandrareddyp